pom.xml

```diff
@@ -1,27 +1,27 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.jboss.windup.plugin</groupId>
-  <artifactId>windup-maven-plugin</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <packaging>maven-plugin</packaging>
+    <groupId>org.jboss.windup.plugin</groupId>
+    <artifactId>windup-maven-plugin</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>maven-plugin</packaging>
 
+    <name>Windup Maven Plugin</name>
 
-  <name>Windup Maven Plugin</name>
+    <url>http://windup.jboss.org</url>
 
-  <url>http://windup.jboss.org</url>
-
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <windup.version>2.4.1.Final</windup.version>
```

Comment:
we probably would like to have here more up-to-date version 2.5.0.CR1/Final or 2.5.0-SNAPSHOT while Final is not ready. What do you think?